plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -1,9 +1,18 @@
 const { chromium, expect } = require( '@playwright/test' );
 const fs = require( 'fs' );
+const {
+	ADMIN_USER,
+	ADMIN_PASSWORD,
+	CUSTOMER_USER,
+	CUSTOMER_PASSWORD,
+} = process.env;
+const adminUsername = ADMIN_USER ?? 'admin';
+const adminPassword = ADMIN_PASSWORD ?? 'password';
+const customerUsername = CUSTOMER_USER ?? 'customer';
+const customerPassword = CUSTOMER_PASSWORD ?? 'password';
 
 module.exports = async ( config ) => {
-	const { stateDir } = config.projects[ 0 ].use;
-	const { baseURL } = config.projects[ 0 ].use;
+	const { stateDir, baseURL, userAgent } = config.projects[ 0 ].use;
```

Comment:
We have to use the same `userAgent` as the one in our Playwright project configuration. Otherwise, our tests would use the default Chromium user agent, which in turn, would cause our nightly smoke test site to throw a `406 Not Acceptable` error.

Reply:
That was the weird issue you were seeing on the Bluehost site, right?